vnext/Shared/Networking/WinRTHttpResource.cpp

```diff
@@ -78,29 +213,28 @@ void WinRTHttpResource::SendRequest(
   try {
     HttpMethod httpMethod{to_hstring(std::move(method))};
     Uri uri{to_hstring(std::move(url))};
-    HttpRequestMessage request{httpMethod, uri};
-
-    auto args = winrt::make<RequestArgs>();
-    auto concreteArgs = args.as<RequestArgs>();
-    concreteArgs->RequestId = requestId;
-    concreteArgs->Headers = std::move(headers);
-    concreteArgs->Data = std::move(data);
-    concreteArgs->IncrementalUpdates = useIncrementalUpdates;
-    concreteArgs->WithCredentials = withCredentials;
-    concreteArgs->ResponseType = std::move(responseType);
-    concreteArgs->Timeout = timeout;
-
-    PerformSendRequest(std::move(request), args);
+
+    auto iReqArgs = winrt::make<RequestArgs>();
+    auto reqArgs = iReqArgs.as<RequestArgs>();
+    reqArgs->RequestId = requestId;
+    reqArgs->Headers = std::move(headers);
+    reqArgs->Data = std::move(data);
+    reqArgs->IncrementalUpdates = useIncrementalUpdates;
+    reqArgs->WithCredentials = withCredentials;
+    reqArgs->ResponseType = std::move(responseType);
+    reqArgs->Timeout = timeout;
+
+    PerformSendRequest(std::move(httpMethod), std::move(uri), iReqArgs);
```

Comment:
>iReqArgs [](http://example.com/codeflow?start=62&length=8)

why move uri and not ireqargs?

Reply:
That was needed for lifetime purposes.
We could probably make it r-valued now.